utils/database.py

```diff
@@ -39,7 +39,7 @@ def setup(self) -> None:
         # Create user_info table and columns to store the user-related variables as a JSON object.
         if "users" not in db:
             pins = db.create_table("users")
-            pins.create_column("user_id", db.types.string)
+            pins.create_column("user_id", db.types.string(100))
```

Comment:
This value here is unnecessary.

Reply:
This value is necessary in order for the user_id column to work

Reply:
But there is no reason one should need to specify the length of the string seeing that it could be varying in any length possible.